tests/test_utils/test_path_helpers.py

```diff
@@ -146,6 +175,27 @@ def test_create_staging_config(config, expected_staging_dict):
     assert staging_dict == expected_staging_dict, "Staging config is not as expected"
 
 
+def test_create_staging_config_berd(berd_config):
+    """Test create_staging_config function for BERD survey type."""
+    expected_staging_dict = {
+        "feather_output": "R:/DAP_emulation/2022_surveys/BERD/01_staging/feather",
+        "snapshot_path": "/2023_snapshots/snapshot_file-202212.json",
+        "updated_snapshot_path": "2023_snapshots/updated_snapshot_file-202212.json",
+        "postcode_masterlist": "postcode_masterlist_path/postcode.csv",
+        "manual_outliers_path": (
+            "R:/DAP_emulation/2022_surveys/BERD/07_outliers/man_out/man_out.csv"
+        ),
+        "manual_imp_trim_path": (
+            "R:/DAP_emulation/2022_surveys/BERD/06_imputation/man_trim/trim_qa.csv"
+        ),
+
+        "backdata_path": "2021_data/backdata.csv",
+    }
+
+    staging_dict = create_staging_config(berd_config)
+
+    assert staging_dict == expected_staging_dict, "Staging config is not as expected"
+
```

Comment:
New test to show that the staging config is correctly set up for BERD as well as PNP (previous test)